website/docs/Software/OpenBCISoftware/01-OpenBCI_GUI.md

```diff
@@ -288,26 +288,42 @@ Expert Mode Keyboard Shortcuts:
 
 **Most users should start by [running the standalone OpenBCI GUI](#running-the-openbci_gui). The Processing IDE is for adding features, modifying existing code, or testing development versions.**
 
-The OpenBCI GUI was built using [Processing](https://processing.org/), a popular, Java-based creative coding framework. If you are interested in adding features or modifying the existing code, it is really easy to do so if you are familiar with Java. If you're not familiar with Java, don't fret! Processing is one of the easiest software packages to pick up as a beginner coder.
+The OpenBCI GUI is built using [Processing 4](https://processing.org/), a popular, Java-based creative coding framework. If you are interested in adding features or modifying the existing code, it is really easy to do so if you are familiar with Java. If you're not familiar with Java, don't fret! Processing is one of the easiest software packages to pick up as a beginner coder.
 
 The things you will need to run the OpenBCI GUI in Processing are:
 
-- [Processing App](https://processing.org/download/?processing)
+- [Processing App](https://processing.org/download)
 - [OpenBCI GUI Sketch](https://github.com/OpenBCI/OpenBCI_GUI)
 
-First, go to processing.org and download the latest version of Processing. [Here is a walkthrough on how to install Processing on Ubuntu Linux.](http://www.artsnova.com/processing/installing-processing-ubuntu-linux-tutorial.html)
+### Installing Processing 4
 
-Go ahead and move it to your `Applications` folder, and launch the application. If this is the first time that you are running Processing, it will create what it calls it's `Sketch` folder in the following locations:
+First, go to Processing's website and [download the latest version of Processing](https://processing.org/download). For more information on Processing, you can review the official [Processing Getting Started Guide](https://processing.org/tutorials/gettingstarted) or the [Processing Documentation](https://processing.org/environment).
 
-On a Mac `Users/<user-name>/Documents/Processing`  
- On Windows `C:\Users\<user-name>\Documents\Processing`  
- On Linux `/home/<user-name>/sketchbook`
+Go ahead and move the Processing application to your `Applications` folder, and launch the application. If this is the first time that you are running Processing, it will create what it calls it's `Sketch` folder in the following locations:
```

Comment:
Optional: `Applications` is a Mac specific folder which may be confusing to Linux and Windows users.

Reply:
Yea, this is legacy wording. I can imagine our users know how to install a program on their desired OS.

Reply:
In that case I would recommend using non-OS specific language. Like "applications" vs "Applications" so people don't think they need to find a folder called "Applications" on Windows.

Reply:
This got replaced with 

Go ahead and install the Processing application as you normally would for your operating system.